Kerdy/Kerdy/InitialSetting/VC/FourthInitialSettingVC.swift

```diff
@@ -163,21 +176,100 @@ final class FourthInitialSettingVC: UIViewController {
         }
     }
     
+    private func setStackView1() {
+        clubStackView1.addArrangedSubview(yappBtn)
+        clubStackView1.addArrangedSubview(soptBtn)
+        clubStackView1.addArrangedSubview(gdscBtn)
```

Comment:
```suggestion
        clubStackView1.addArrangedSubviews(yappBtn, soptBtn, gdscBtn)
```
extension으로 addArrangedSubviews 추가해뒀습니다 :)